pages/ParentTips.js

```diff
@@ -44,8 +45,14 @@ export const ParentTips = ({ navigation }) => {
         <ParentTipsContainer>
             <GeometryBackground />
             <ParentTipsHeaderContainer>
-                <ParentTipsHeader>
-                    Interpret Your Child's Reactions
+                <ParentTipsHeader
+                    style={{
+                        fontFamily: 'Avenir',
+                        fontWeight: 'bold',
+                        fontSize: '35'
+                    }}
```

Comment:
please dont use in-line styling. Just make the change on ParentTipsHeader

Reply:
The font-size, according to the styling, should be 24px. ParentTipsHeader is already 25px. You can just remove this style.